### ISSUE TITLE
How to pass dictionary(array) GET parameter?

### ISSUE THREAD

**Issue by pocket7878** [ishkawa/APIKit#75]
Some API requests parameter with array format with GET method.

```
some.api.com?data[id]=10
```

I tried to set parameter like:

```swift
var parameters: [String: AnyObject] {
   return ["data[id]": 10]
}
```

and

```swift
var parameters: [String: AnyObject] {
   return ["data": ["id": 10]]
}
```

Both throws error

```
NSInvalidArgumentException', reason: '*** +[NSURLComponents setPercentEncodedQuery:]: invalid characters in percentEncodedQuery
```

Maybe parameter contains invalid characater '[' and ']'.

I should escape parameter key myself or any easy way to build such parameter?

**Comment by pocket7878** [comment]
I just created very simple PR to escaping GET parameter key.
https://github.com/ishkawa/APIKit/pull/76

**Comment by ikesyo** [comment]
Resolved by #76.
